study.md

```diff
@@ -66,7 +66,7 @@ Which ActiveRecord method finds all of the records of a certain type (or
 entity)?
 
 ```md
-<!-- your response here -->
+'where'.  For example: users = User.where(name: 'Joe')     will find all the users who have the name 'Joe'
```

Comment:
`#all` will retrieve every record of a certain type.  `#where` will do the same but only if no filtering arguments are passed into it.